ePBFT/src/pbft/libbyz/keypair.h

```diff
@@ -128,9 +128,9 @@ class KeyPair
    */
   std::vector<uint8_t> sign(unsigned char* d, size_t d_size) const
   {
-    uint8_t sig[max_sig_size];
-    sign(d, d_size, sig);
-    return {sig, sig + signature_size};
+    std::vector<uint8_t> sig(max_sig_size);
```

Comment:
Let's leave this as it was as `libbyz/keypair.h` and `tls/keypair.h` will have to be merged at some point.

Reply:
I had changed this because the sanitizer was complaining when in Debug mode, but I don't think it's needed actually. I have reverted it.